addon/src/motions/adjust-css.ts

```diff
@@ -1,6 +1,7 @@
-import Motion from '../-private/motion';
-import Tween from '../-private/tween';
+import Motion, { BaseOptions } from '../-private/motion';
```

Comment:
@wagenet should we do here also `import Motion, { type BaseOptions } from '../-private/motion';` as well as everywhere else in the PR?